.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,178 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests" ] #REMOVE WHEN ON MASTER
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+(.alpha.[0-9]+)*"
+  pull_request_review:
+    types: [submitted]
+  workflow_dispatch:
+    inputs:
+      bob_nodes_amount:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_vdisk_count:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas_amount:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload_size:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_record_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '10000'
+
+jobs:
+  build-binaries:
+    if: ${{ github.event.review.state == 'approved' || (github.event_name != 'pull_request_review' && github.event_name != 'push') }}
```

Comment:
Why did you discarded `push` events?
Also the condition is not good: you can check for review state only if it is a `pull_request_review` event. 
So the condition should be something like this: `github.event_name != 'pull_request_review' || github.event.review.state == 'approved'`